### DISCUSSION TITLE
How do I get reports to work with newest version

### DISCUSSION THREAD

**Discussion by jadams3710** [netbox-community/netbox#8459] [Q&A] [answered]
We just did a big jump with NetBox from 2.10.6 to 3.1.6 , when I click on reports I get a server Error screenshot below.
Can anyone tell me what I have to remove from a previous configuration or anything to get reports to work?
Thanks in advance
![image](https://user-images.githubusercontent.com/55246184/151191599-165bfa9a-d35c-4fcf-b572-55f8e0e1d5df.png)

**Comment by jeremystretch** [comment, marked as answer]
A complete accounting of major changes is provided in the [release notes](https://netbox.readthedocs.io/en/stable/release-notes/) of each release. This particular error stems from the RackGroup model being renamed to Location in v2.11, however if you haven't read any of the release notes before upgrading you'll likely run into other problems.

**Reply by jadams3710** [reply]
Thank you for the help I have been slowly reading through changes but since we did such a big jump it is taking awhile.
